src/qtt/qiskit/circuits.py

```diff
@@ -0,0 +1,45 @@
+from typing import List, Optional
+
+from qiskit.circuit.quantumcircuit import QuantumCircuit
+
+
+def integrate_circuit(
+    qc: QuantumCircuit,
```

Comment:
What happens when qc exists of more than 1 groups of qregs?

Reply:
That is no problem. I added a test case to be sure